### PR TITLE
move contact requests & installations to protobuf

### PR DESCRIPTION
**Pull request by cammellos**
This commits moves from transit contact updates/installations to protobuf.

After this commit transit can be removed, this PR might not remove the code until successfully tested (might be done in a different PR).

status: ready

### CODE REVIEW

**Review by yenda — APPROVED** [review]

File: src/status_im/contact/core.cljs [hunk 1]
```diff
@@ -51,6 +54,27 @@
...
+              (multiaccounts.update/multiaccount-update :last-updated last-updated {:dont-sync? true})
+              (multiaccounts.update/multiaccount-update :photo-path photo-path {:dont-sync? true}))))
+
+(fx/defn ensure-contact
+  [{:keys [db] :as cofx}
+   {:keys [public-key] :as contact}]
+  (let [new? (get-in db [:contacts/contacts public-key])
+        us? (= public-key (multiaccounts.model/current-public-key cofx))]
+    (fx/merge cofx
+              {:db (-> db
```
Comment (yenda) [inline src/status_im/contact/core.cljs:69]: threading macro is superfluous here

File: src/status_im/multiaccounts/update/core.cljs [hunk 2]
```diff
@@ -58,7 +33,8 @@
                  [{:method "settings_saveSetting"
                    :params [setting setting-value]
                    :on-success on-success}]}
-                (when (#{:name :photo-path :prefered-name} setting)
+                (when (and (not dont-sync?)
```
Comment (yenda) [inline src/status_im/multiaccounts/update/core.cljs:36]: not dont-sync? :D

Comment (cammellos) [reply src/status_im/multiaccounts/update/core.cljs:36]: yes, is a bit silly, but I need a negative, as 99% of the time we want to sync, only when reacting to a syncing event we don't want to sync :) anything better we could name it?

**Comment by churik** [comment]
@cammellos 
### ISSUE 1: User appears as contact of himself after changing userpic

**Steps:**
- install status
- create new multiaccount
- go to profile and change picture
- reopen app
- check contact list

**Expected result:** contact list is empty
**Actual result:** user is added as contact of himself
**Fleet:** eth.staging

Testing is in progress, so more comments/issues will be added

**Comment by churik** [comment]
@cammellos would you mind to rebase your pr to include https://github.com/status-im/status-react/pull/9808 ?
I'm stuck when checking https://ethstatus.testrail.net/index.php?/tests/view/1141312 - couldn't reproduce it manually, so feeling like e2e build is not updated.

**Comment by cammellos** [comment]
@churik rebased, it should also fix issue 1, although I hadn't time to test the build.
Thanks!

**Comment by cammellos** [comment]
@churik I could not replicate . test_pair_devices_sync_name_photo_public_group_chats , although the test seems to be consistently failing, did you manage to replicate? (confirmed issue 1 is fixed )

**Comment by churik** [comment]
@cammellos nope, I didn't on android builds. I'll run them locally and investigate this, however it shouldn't block PR from merging.
Let you know when I finish with testing

**Comment by churik** [comment]
Tested:
- pairing of three devices on fresh account
- syncing of public / 1-1/ group chats and profile picture
- syncing of messages when device came offline / online
- changing profile photo
- ENS name in profile (all issues are in nightly, didn't find something PR specific)

There is issue with pairing when one of the devices has custom time, but this one can be fixed separately.

**Pull request merged**